classes/Link.php

```diff
@@ -730,6 +730,22 @@ public function getAdminLink($controller, $withToken = true, $sfRouteParams = ar
                 }
                 break;
 
+            case 'AdminTranslations':
+                // In case of email body translations we want to get a link to legacy controller,
+                // in other cases - it's the migrated controller
+                if (isset($params['selected-emails']) && $params['selected-emails'] === 'body') {
```

Comment:
Hello @rokaszygmantas , does it means that there is still work to do on Translations page?

Can you sum up what actions/task needs to be migrated please?

Reply:
The legacy controller is still used in some cases to access the translations page (the one where you modify the translations). For example, some modules are still using the old translation system and for those it's still loading the legacy controller instead of the translations app. Also, if you select email body translations it will still load the legacy controller. I'm not sure why it is working like that, so didn't want to break all the translations by changing this behavior. :) maybe someone else from the core team knows?

Reply:
ok, fair enough, this is something i'll have to discuss with product team :)